sources/ElkArte/Database/QueryInterface.php

```diff
@@ -66,7 +66,7 @@ public function query($identifier, $db_string, $db_values = array());
 	 * @param string $db_string
 	 * @param mixed[] $db_values = array()
 	 * @param mixed[]|null
-	 * @return bool|Result
+	 * @return bool|\ElkArte\Database\AbstractResult
```

Comment:
I wonder if I lost a ResultInterface somewhere or if I left it out on purpose...